docker/smqtk_services.run_images.sh

```diff
@@ -38,57 +30,53 @@ done
 # Argument error checking
 if [ -z "${IMAGE_DIR}" ]
 then
-    echo "ERROR: No image directory provided. Please provide -i|--images option."
+    (
+        echo -n "ERROR: No image directory provided."
+        echo -n " Please provide -i|--images option."
+        echo
+    ) >&2
+
     exit 1
 elif [ -z "$(ls "${IMAGE_DIR}")" ]
 then
-    echo "ERROR: Nothing in provided image directory: ${IMAGE_DIR}"
+    echo "ERROR: Nothing in provided image directory: ${IMAGE_DIR}" >&2
     exit 2
 fi
 
-if [ -z "${DOCKER_NETWORK}" ]
-then
-    echo "ERROR: No docker network name given"
-    exit 3
-fi
-
-#
-# Make sure docker network exists / is created
-#
-if [ -z "$(docker network ls 2>/dev/null | tail -n +2 | grep "${DOCKER_NETWORK}")" ]
-then
-    # network doesn't exist yet, create it
-    echo "Creating new docker network \"${DOCKER_NETWORK}\" using bridge driver"
-    docker network create --driver bridge "${DOCKER_NETWORK}"
-else
-    echo "Found docker network \"${DOCKER_NETWORK}\" already exists"
-fi
-
-#
-# Start and Initialize PostgreSQL container
-#
-echo "Starting up PostgreSQL docker"
-docker run -d --name "${DOCKER_POSTGRES}" --net=${DOCKER_NETWORK} postgres
+export IMAGE_DIR
+mkdir -p smqtk_logs
+docker-compose up -d postgres
 
 # Wait until PSQL instance is up and running by poking psql
-echo "Waiting for a responsive database"
-q=""
 trigger="DatabaseNowResponsive"
-while [ -z "$q" ]
-do
+poll_command='docker-compose exec postgres psql postgres postgres'
+poll_command="$poll_command -c \"\\\\echo $trigger\" 2>/dev/null"
+poll_command="$poll_command | grep -q \"$trigger\""
+
+echo "Waiting for a responsive database"
+(
     set +e
-    q="$(docker exec ${DOCKER_POSTGRES} psql postgres postgres -c "\echo ${trigger}" 2>/dev/null | grep "${trigger}")"
```

Comment:
Though it might not be strictly necessary, the "officially sanctioned" way to do these sorts of things are using the `wait-for-it` script: https://github.com/vishnubob/wait-for-it